examples/hello-openshift/hello-project.json

```diff
@@ -1,10 +1,10 @@
 {
   "apiVersion": "v1beta1",
-  "displayName": "Hello OpenShift",
   "kind": "Project",
   "metadata": {
     "annotations": {
-      "description": "This is an example project to demonstrate OpenShift v3"
+      "description": "This is an example project to demonstrate OpenShift v3",
+      "displayName": "Hello OpenShift"
```

Comment:
We should not change the v1beta1 syntax for the file.

Create a copy of this file, and put in /examples/hello-openshift/v1beta2/hello-project.json.  Change the version of the file to v1beta2 from v1beta1